packages/sitecore-jss-react/src/components/FEaaSComponent.test.tsx

```diff
@@ -166,5 +166,27 @@ describe('<FEaaSComponent />', () => {
         `data="${props.params?.ComponentDataOverride!.replace(/"/g, '&quot;').replace(/\s/g, '')}"`
       );
     });
+
+    it('should get data using SSR', () => {
```

Comment:
You need to test a case when you provide _params.ComponentDataOverride_ but prefetch data on the server side

Reply:
Yes that makes sense so I have added this scenario in this particular test case itself as we already have a test above when there is no prefetch data but we have `params.ComponentDataOverride`.

Reply:
Makes sense to me!